README.md

```diff
@@ -123,23 +122,25 @@ You can turn off SIP on El Capitan this way:
   * Click the  (Apple) menu and select Restart....
 
 The following commands are used to bootstrap CodeChecker on
-OS X El Capitan 10.11.5 and macOS Sierra 10.12 Beta.
-
-Check out and build LLVM/Clang with extra tools. Follow the [Get Started with
-LLVM/Clang](http://clang.llvm.org/get_started.html) documentation.
+OS X El Capitan 10.11, macOS Sierra 10.12 and macOS High Sierra 10.13.
 
 ~~~{.sh}
 # Download and install dependencies.
 brew update
 brew install doxygen gcc git
-brew install homebrew/versions/thrift090
+brew llvm
```

Comment:
The `clang` compiler is installed by default. Do we need the `llvm` here?

Reply:
Ok, I see, `llvm` will install the `clang-tools-extra` where clang tidy is. In that case the latest points to llvm 5.0 I think we should be in sync with the available clang version. I'm not sure what is the latest available clang version in OSX right now. In Travis I think we should use the same `clang` and `clang-tidy` versions.

Reply:
Does this mean it is ok to merge this as is?

Reply:
@gyorb ```Apple LLVM version 9.0.0 (clang-900.0.38)``` from Xcode 9 is based on clang 4.0.

At the moment I am not able to check which version will be installed with ```brew install llvm``` but I will as soon as possible.

Reply:
Hmm, maybe it would be better to force the version of LLVM somehow, similar to thrift. The problem is, we do not want to break the TravisCI when the llvm package is upgraded in the repository.